README.md

```diff
@@ -419,10 +419,12 @@ self-hosted Huly, perform the following steps:
       front:
         ...
         environment:
-          - LIVEKIT_WS=<LIVEKIT_HOST>
+          - LIVEKIT_WS=ws${SECURE:+s}://<LIVEKIT_HOST>
```

Comment:
Hi @acheong08 
`LIVEKIT_HOST` should include protocol as well, so I suggest to add a note about it instead of this change.
For example, it is used to create Livekit clients and the protocol is required according to the Livekit docs:
https://docs.livekit.io/reference/server-sdk-js/classes/roomserviceclient.html